src/platforms/dart/configuration/integrations/http-integration.mdx

```diff
@@ -49,8 +64,13 @@ var uriResponse = await client.post('https://example.com/whatsit/create',
 
 ## Performance Monitoring for HTTP Requests
 
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
```

Comment:
```suggestion
Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>.
```

Reply:
I'd be fine changing it but this sentence is in ~15 different places, maybe a separate PR addressing all of them?